pandas/core/indexes/base.py

```diff
@@ -3661,7 +3663,9 @@ def _wrap_difference_result(self, other, result):
         # We will override for MultiIndex to handle empty results
         return self._wrap_setop_result(other, result)
 
-    def symmetric_difference(self, other, result_name=None, sort=None):
+    def symmetric_difference(
+        self, other, result_name: str | None = None, sort: bool | None = None
```

Comment:
```suggestion
        self, other, result_name: Hashable  | None = None, sort: bool | None = None
```

Okay, made the change